docs/_docs/configuration/sass.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Sass/SCSS Options
+permalink: "/docs/configuration/sass/"
+---
+
+By default, Jekyll looks for Sass partials in the `_sass` directory relative to your site's `source` directory. You can change the default load path or specify additional load paths with the following options:
+
+* **sass_dir** - A filesystem-path which should be searched for Sass partials.
+* **load_paths** - An array of additional filesystem-paths which should be searched for Sass partials.
+
+<div class="note info">
+  <p>
+    Note that filesystem-paths specified in the <code>sass_dir</code> and
+    <code>load_paths</code> options are resolved relative to your site's
+    <code>source</code>, not relative to the location of the
+    <code>_config.yml</code> file.
+  </p>
+</div>
+
+The following additional Sass configuration options are offered by the [jekyll-sass-converter](https://github.com/jekyll/jekyll-sass-converter) plugin.
```

Comment:
This is not core Jekyll - maybe leave this out and rather link to that plugin's docs?

When the plugin changes, these docs become out of date, so rather just provide a reference.